imports/ui/PlayerRoom.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Presence } from "meteor/tmeasday:presence";
+import { GamesCollection } from "../api/games";
+import { useTracker } from "meteor/react-meteor-data";
+
+// put the player into the gameID (game room) after pressing play(done)
+// dont show the list for both playing players(done)
+// dont show users if they are in the game for other players(done)
+// make a quit game button ant after click set status back to isPlaying: false(done)
+// and show them the waiting room(done)
+
+const Room = ({ room }) => {
+  const myName = Presence.state("user");
+
+  const [hideList, setHideList] = useState(false);
+  const [quit, setQuit] = useState(false);
+
```

Comment:
Are these always the opposite? Can you just use one state here and infer the other one? Something 
```suggestion
  const [quit, setQuit] = useState(false);
  const hideList = !quit;
```